packages/@angular/cli/commands/help.ts

```diff
@@ -55,7 +55,13 @@ const HelpCommand = Command.extend({
       });
 
       if (rawArgs.length > 0) {
-        if (cmd === rawArgs[0]) {
+        let commandInput = rawArgs[0];
+        const alias = Command.prototype.aliases && Command.prototype.aliases[0];
+        if (commandInput.length === 1 && commandInput === alias) {
```

Comment:
We don't need to iterate just check alias passed and current command are same. Removed check for length 1.